gatorgrade/main.py

```diff
@@ -0,0 +1,35 @@
+"""Use Typer to run gatorgrade to run the checks and generate the yml file."""
+from pathlib import Path
+import typer
+
+app = typer.Typer(add_completion=False)
+FILE = "gatorgrade.yml"
+
+
+@app.callback(invoke_without_command=True)
+def gatorgrade(
+    filename: Path = typer.Option(
+        f"{FILE}", "--config", "-c", help="Name of the yml file."
+    )
+):
+    """Run the GatorGrader checks in the gatorgrade.yml file."""
+    if filename.contains(".yml"):
```

Comment:
This line currently throws the error, `Result AttributeError("'PosixPath' object has no attribute 'contains'")`, as `filename` is of type [Path](https://docs.python.org/3/library/pathlib.html), which does not have a `contains` method. Instead, what if you used the [`suffix` attribute](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix) to check the extension?